### ISSUE TITLE
Lists don't contain only <li> supporting elements & script supporting elements 

### ISSUE THREAD

**Issue by mkatenolan** [Coletterbox/coletterbox#3]
See more [here](https://dequeuniversity.com/rules/axe/3.1/list?application=lighthouse) ...

Screen readers have a specific way of announcing lists. This feature makes lists clearer to understand, but will only work if lists are properly structured.

When content elements other than list items are contained within a set of list elements, screen readers cannot inform the listener that they are listening to items within the list.

For a list to be valid, it must have both parent elements (a set of ul elements or a set of ol elements) and child elements (declared inside of these tags using the li element), and any other content elements are invalid.

Although some non-content elements such as script, template, style, meta, link, map, area, and datalist are permitted within lists, content elements other than li are not permitted.
